src/platform/packages/shared/kbn-ebt-tools/src/performance_metrics/context/use_performance_context.tsx

```diff
@@ -15,6 +15,19 @@ export interface PerformanceApi {
    * @param eventData - Data to send with the performance measure, conforming the structure of a {@link EventData}.
    */
   onPageReady(eventData?: EventData): void;
+  /**
+   * Marks the start of a page refresh event for performance tracking.
+   * This method adds a performance marker start::pageRefresh to indicate when a page refresh begins.
+   *
+   * Usage:
+   * ```ts
+   * markPerformanceRefreshStart();
+   * ```
+   *
+   * The marker set by this function can later be used in performance measurements
+   * along with an end marker end::pageReady to determine the total refresh duration.
+   */
+  markPerformanceRefreshStart(): void;
```

Comment:
```suggestion
  onPageRefreshStart(): void;
```
To be consistent with `onPageReady` and to abstract the performance marking bit. Not a strong opinion though.

Reply:
Thanks for the suggestion. I will update